### PR TITLE
added /tick (in new 1.20.3 file)

### PR DESCRIPTION
**Pull request by InQuognito**

### CODE REVIEW

**Review by ChenCMD — APPROVED** [review]
LGTM

File: src/data/CommandTree1.20.3.ts [hunk 1]
```diff
@@ -0,0 +1,3667 @@
...
+                        rate: {
+                            parser: new NumberArgumentParser('float', 1, 10_000),
+                            executable: true
+                        }
+                    }
+                },
+                sprint: {
+                    parser: new LiteralArgumentParser('sprint'),
+                    children: {
+                        [Switchable]: true,
```
Comment (ChenCMD) [inline src/data/CommandTree1.20.3.ts:2610]: Do not add Switchable when all children are not LiteralArgument.
```suggestion
```

File: src/data/CommandTree1.20.3.ts [hunk 2]
```diff
@@ -0,0 +1,3667 @@
...
+                        time: {
+                            parser: new TimeArgumentParser(),
+                            executable: true
+                        }
+                    }
+                },
+                step: {
+                    parser: new LiteralArgumentParser('sprint'),
+                    children: {
+                        [Switchable]: true,
```
Comment (ChenCMD) [inline src/data/CommandTree1.20.3.ts:2624]: Same as sprint.
and forgot to change.
```suggestion
                step: {
                    parser: new LiteralArgumentParser('step'),
                    children: {
```

**Review by ChenCMD — CHANGES_REQUESTED** [review]
We checked the operation again and found a problem.

**Comment by misode** [comment]
Merging so Chen can continue this

**Pull request merged**